### ISSUE TITLE
PDF Viewer in 53 does not work in WPF application

### ISSUE THREAD

**Issue by happyuk** [cefsharp/CefSharp#1858]
###Bug Report

- **What version of the product are you using?**
    -53

- **What architecture x86 or x64?**
   - Either

- **On what operating system?**
    - Win7

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
  - Wpf

- **What steps will reproduce the problem?**
    
1. Download and compile the 'minimal' example:
https://github.com/cefsharp/CefSharp.MinimalExample

2. Set the 'Address' property in the MainWindow.xaml to point to a pdf file

(This is the only code change I have made to this example)

```
<wpf:ChromiumWebBrowser Grid.Row="0"
                            x:Name="Browser"
                            Address="http://www.cbu.edu.zm/downloads/pdf-sample.pdf" />
```

3. Re-compile and run

- **What is the expected output? What do you see instead?**

The PDF document does not materialize until the Wpf app is re-sized or maximized.

Once you do re-size, the PDF document becomes visible, but you cannot use any of the controls within it without ALSO doing an additional resize action eg + / - / use the scroll bar etc.

- **Please provide any additional information below.**

Seems very similar to a previous issue raised and closed:

https://github.com/cefsharp/CefSharp/issues/1697

- **Does this problem also occur in the `Cef` TestApp from http://opensource.spotify.com/cefbuilds/index.html (http://cefbuilds.com for older version)?**

This site is currently blocked by my employer (lumps the spotify bit in with 'Music') but I will try to have a look when I get the chance.

**Comment by mschleeweiss** [comment]
I have the same issue. It also seems to work (but with very poor performance) if the dev tools are visible (`browser.ShowDevTools();`)

**Comment by happyuk** [comment]
Hi Marc thanks for your input do you know of a version that does not have
this problem in wpf?

**Comment by amaitland** [comment]
Duplicate of #1799
